javascript/javascript1/week1/homework_soheib/houseyPricey.js

```diff
@@ -0,0 +1,40 @@
+let peterHouse = {
+    width: 8,
+    depth: 10,
+    height: 10,
+    garden: 100,
+    price: 2500000
+};
+
+let juliaHouse = {
+    width: 5,
+    depth: 11,
+    height: 8,
+    garden: 70,
+    price: 1000000
+};
+
+function volumeCalc(house){
+    let volume = house.depth * house.width * house.height
+    return volume;
+}
+
+function houseValueCalc(house){
+    let houseValue = volumeCalc(house) * 2.5 * 1000 + house.garden * 300;
```

Comment:
all of these can be declared const, or you can also just return the calculation itself 👍

Reply:
That is a much better way. Many thanks